lib/puppet/configurer.rb

```diff
@@ -358,7 +319,7 @@ def run_internal(options)
 
       query_options, facts = get_facts(options) unless query_options
       query_options[:configured_environment] = configured_environment
-      options[:convert_for_node] = node
+      # options[:convert_for_node] = node
```

Comment:
`node` does not exist anymore, also I can't see this option being used anywhere so I think we're safe to remove the line entirely?

Reply:
Yeah it was added in 29d515761eab068381900c74bf8c4b6165853fa9, eliminated in 091dba40812d18658485d47231a576ad331ccee8, but this reference is still there. :+1: to deleting it.